_commands/toys/cowsay.md

```diff
@@ -2,23 +2,111 @@
 ---
 
 cowsay
--------
+--
+`cowsay` generates an ASCII picture of a cow saying something provided by  the  user.
 
-ToDo: Needs Completion
 
 ~~~ bash
-*short* example of command usage and output
+$ cowsay Hi There
+ __________ 
+< Hi There >
+ ---------- 
+        \   ^__^
+         \  (oo)\_______
+            (__)\       )\/\
+                ||----w |
+                ||     ||
 ~~~
 
 <!--more-->
 
+
+
 ### Useful Options / Examples
+There are many useful options for this intersting cowsay command.
+The complete list is: cowsay  [-e  eye_string] [-f cowfile] [-h] [-l] [-n] [-T tongue_string] [-W column] [-bdgpstwy]
+
+-e allows the user to customize the cow's eyes which are defaulted to "oo" as seen above.
+-T allows the user to add a tongue string to the cow image. By default there is no tongue string.
```

Comment:
Everything looks great, but I have one request ... This may seem a bit nit-picky, but I swear its worth it for the consistency. Can you surround every flag or `[option]` with backtics \`\`?